src/makeRobot.js

```diff
@@ -38,7 +39,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords:
+    {
+      x: 0, y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.y += steps;
+
+      return this;
+    },
+
+    goBack(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.y -= steps;
+
+      return this;
+    },
+
+    goRight(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.x += steps;
+
+      return this;
+    },
+
+    goLeft(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.x -= steps;
+
+      return this;
+    },
```

Comment:
```suggestion
    goLeft(steps = 1) {
      if (steps > 0) {
        this.coords.x -= steps;
      }
      
      return this;
    },
```

Let`s remake every method like this to make you code looks more pretty